account.js

```diff
@@ -1,17 +1,48 @@
+import open from 'open'
+import { confirm } from '@inquirer/prompts'
 import * as Account from '@web3-storage/w3up-client/account'
 import * as Result from '@web3-storage/w3up-client/result'
 import * as DidMailto from '@web3-storage/did-mailto'
+import { authorize } from '@web3-storage/capabilities/access'
+import { base64url } from 'multiformats/bases/base64'
 import { getClient } from './lib.js'
 import ora from 'ora'
 
 /**
  * @typedef {Awaited<ReturnType<Account.login>>['ok']&{}} View
  */
 
+const OAuthProviderGitHub = 'github'
+const OAuthProviders = /** @type {const} */ ([OAuthProviderGitHub])
+
+/** @type {Record<import('@web3-storage/w3up-client/types').DID, string>} */
+const GitHubOauthClientIDs = {
+  'did:web:web3.storage': 'Ov23liRdyizj8EndxxAf',
+  'did:web:staging.web3.storage': 'Ov23liRdyizj8EndxxAf',
```

Comment:
Is it OK to use the same id for `prod` & `staging` in this case?

Reply:
I still need to setup the apps. Thats my test oauth app.